frontend/packages/console-dynamic-plugin-sdk/docs/console-extensions.md

```diff
@@ -1218,18 +1218,18 @@ Topology relationship provider connector extension
 
 | Name | Value Type | Optional | Description |
 | ---- | ---------- | -------- | ----------- |
-| `provides` | `CodeRef<RelationshipProviderProvides>` | no |  |
-| `tooltip` | `string` | no |  |
-| `create` | `CodeRef<RelationshipProviderCreate>` | no |  |
-| `priority` | `number` | no |  |
+| `provides` | `CodeRef<RelationshipProviderProvides>` | no | use to determine if a connection can be created between the source and target node |
+| `tooltip` | `string` | no | tooltip to show when connector operation is hovering over the drop target ex: "Create a Visual Connector" |
+| `create` | `CodeRef<RelationshipProviderCreate>` | no | callback to execute when connector is drop over target node to create a connection |
+| `priority` | `number` | no | priority for relationship, higher will be preferred in case of multiple |
 
 ---
 
 ## `console.user-preference/group`
 
 ### Summary 
 
-(not available)
+Allows to add a group on the console user preferences page. Which will appears as a vertical tab option on the console user-preferences page.
```

Comment:
```suggestion
This extension can be used to add a group on the console user preferences page. It will appear as a vertical tab option on the console user-preferences page.
```